integration_tests/pages/page.ts

```diff
@@ -117,4 +117,22 @@ export default abstract class Page {
   clickLink(label: string): void {
     cy.get('a').contains(label).click()
   }
+
+  shouldShowPrintButton(): void {
+    cy.get('button').contains('Save as PDF')
+  }
+
+  clickPrintButton(): void {
+    cy.get('button').contains('Save as PDF').click()
+  }
+
+  shouldPrint(): void {
+    cy.window().then(win => {
+      cy.stub(win, 'print').as('printStub')
+    })
+
+    this.clickPrintButton()
+
+    cy.get('@printStub').should('be.calledOnce')
+  }
```

Comment:
do you think this change would be a bit clearer in 2 commits, e.g.

1. refactor: move page test helpers from `SubmittedApplicationPage` to `Page`
2. change in behaviour: add print button to referrer's view of submitted application

Reply:
yes, good shout